src/figuresClasses.ts

```diff
@@ -1,11 +1,101 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: `red` | `green` | `blue`;
 
-export class Triangle implements Figure {}
+  getArea(): number;
+}
+export class Triangle implements Figure {
+  shape: 'triangle';
```

Comment:
1) u can assign the value u want right away instead of doing it at the end
2) use the `union type` u created at the beginning
```suggestion
  shape: Shape;
```